### DISCUSSION TITLE
Cannot see private repos by default

### DISCUSSION THREAD

**Discussion by jzombie** [community/community#37423] [Other Feature Feedback, Questions, & Ideas] [answered]
Some new GitHub updates must have been released which prevents private repositories from being shown by default.  Is there a way to re-enable this feature?  Otherwise, it seems if I don't recall the name of a private repo, it could be "lost."

**Comment by jzombie** [comment, marked as answer]
I discovered a workaround...  Click "type" next to "Find a repository," select "Public," then select "All".  It's a bit inconvenient, though.

![Screen Shot 2022-10-26 at 9 30 15 PM](https://user-images.githubusercontent.com/1139657/198184758-140da2a2-4a3b-4cdd-95df-561d369695b5.png)

**Reply by piotrekkr** [reply]
If you try to see organization private repositories you could also change "View as" to member
![image](https://user-images.githubusercontent.com/503782/198510701-a8aae13a-0eb7-4881-8712-c492a528eeaa.png)
Both of those solution do not work for long since after page reload or tab reopen those setting are lost... Bad UX :disappointed:

**Reply by jzombie** [reply]
Thanks @piotrekkr.  I also noticed that clicking the "repositories" link at the top displays the private repos by default.


![Screenshot from 2022-10-28 12-16-32](https://user-images.githubusercontent.com/1139657/198705031-6d30c29f-d687-4a05-bb56-1e5dbd7bc9df.png)

**Reply by piotrekkr** [reply]
@jzombie Yeah, but still so inconvenient... Someone at GH should really think on UI/UX so I don't need to use Refined GitHub  browser add-on to fix UX/UI problems :/

**Reply by jzombie** [reply]
@piotrekkr, I agree completely.

**Reply by kongdd** [reply]
I have the same need. Do you have solution?

**Reply by jzombie** [reply]
@kongdd Outside of the workarounds described above, no.
